package.json

```diff
@@ -107,6 +107,7 @@
     "lru-cache": "^6.0.0",
     "measured-reporting": "^1.51.1",
     "module-details-from-path": "^1.0.3",
+    "mongoose": "^7.2.2",
```

Comment:
Oh, oops, this is in "dependencies" instead of "devDependencies". That needs to change.

Reply:
My bad. maybe in one of the tests I've used ' --save` instead of `--save-dev`

Reply:
Fixed!